src/renderer.js

```diff
@@ -113,8 +115,15 @@ Renderer.prototype = {
   },
   
 
-  render: function(svg, collection, tilePoint) {
+  render: function(svg, data, tilePoint) {
     var self = this;
+    var collection;
+    if(data.type === "Topology"){
```

Comment:
I think the renderer should get the data in the format needed to be rendered, is there a reason why we are doing this here?